src/Tokenizers/PHP.php

```diff
@@ -589,6 +589,58 @@ protected function tokenize($string)
                 echo PHP_EOL;
             }
 
+            if ($tokenIsArray === true
```

Comment:
A comment above here to quickly see what this block is about would help.

Reply:
Added.